dashboard/test/ui/features/applab/scenarios.feature

```diff
@@ -37,3 +37,15 @@ Feature: App Lab Share
     And element "#codeModeButton" is hidden
     And element "#designModeButton" is hidden
     And element "#viewDataButton" is hidden
+
+  @no_mobile
+  Scenario: App Lab Http Image
+
+    # Create an app with an http image.
+    Then I press "show-code-header"
+    And I wait to see Droplet text mode
+    And I press keys "image('test123', 'http://example.com" for element ".ace_text-input"
```

Comment:
It ends up testing autocomplete as well, but I can add the closing `')` since we already have autocomplete coverage elsewhere.